src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail.html

```diff
@@ -19,12 +19,16 @@
     <div class="kd-replicationcontrollerdetail-sidebar-header-wrapper kd-replicationcontrollerdetail-sidebar-item"
         flex="nogrow">
       <div class="kd-replicationcontrollerdetail-sidebar-header" layout-align="start center" layout="row">
-        <md-button class="md-icon-button" ui-sref="replicationcontrollers">
-          <md-icon md-font-library="material-icons">arrow_back</md-icon>
-        </md-button>
-        <h1 flex class="md-title kd-replicationcontrollerdetail-app-name">
-          {{::ctrl.replicationControllerDetail.name}}
-        </h1>
+        <div class="kd-replicationcontrollerdetail-sidebar-back-button" flex="10">
+          <md-button class="md-icon-button" ui-sref="replicationcontrollers">
+            <md-icon md-font-library="material-icons">arrow_back</md-icon>
+          </md-button>
+        </div>
+        <span class="md-title kd-replicationcontrollerdetail-app-name"
+              flex="85">
```

Comment:
All flexing is done to restrict elements from becoming too big and overlapping other elements. I'll check if this is still needed because I've done it like 2 iterations before, but I'm pretty sure that it is.